app/utils/encryptable_transfer.rb

```diff
@@ -28,6 +30,48 @@ def receive(encryptable, private_key, personal_team_password)
 
   private
 
+  def encryptable_destination_name(encryptable_name, receiver)
+    inbox_folder_names = receiver.inbox_folder.encryptables.pluck(:name)
+    counter = 0
+
+    loop do
+      matching_inbox_names = find_matching_inbox_names(encryptable_name, inbox_folder_names)
+      break if matching_inbox_names.empty?
+
+      # rubocop:disable Metrics/LineLength
+      new_encryptable_name = generate_new_encryptable_name(encryptable_name, counter, matching_inbox_names)
+      # rubocop:enable Metrics/LineLength
+
+      encryptable_name = new_encryptable_name
+      counter += 1
+    end
+    encryptable_name
+  end
+
+  def generate_new_encryptable_name(encryptable_name, counter, matching_inbox_names)
+    clip_regex = / \(\d+\)/
```

Comment:
```suggestion
    INCREMENT_REGEX = / \(\d+\)/
```

also consider putting constants on top of the class ... like in java :wink: